docs/installation/centos.md

```diff
@@ -58,9 +58,10 @@ There are two ways to install Docker Engine.  You can use `curl` with the  `get.
 		
 		This script adds the `docker.repo` repository and installs Docker.
 
-4. Start the Docker daemon.
+4. Start the Docker daemon and enable it so it starts on future boots.
 
-		$ sudo service docker start
+		$ sudo systemctl start docker
+		$ sudo systemctl enable docker
```

Comment:
How is this different that "Start the docker daemon at boot" this heading down the page?  Why is the command different. I'd prefer to move the start on boot to its own section...